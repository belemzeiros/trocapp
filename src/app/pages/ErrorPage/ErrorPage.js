import React from 'react';
import PropTypes from 'prop-types';
import { Helmet } from 'react-helmet';

const ErrorPage = ({ staticContext }) => {
  staticContext.status = 404;

  return (
    <React.Fragment>
      <Helmet>
        <title>Erro página não encontrada</title>
      </Helmet>
      <div>Página não existe!</div>
    </React.Fragment>
  );
};

ErrorPage.defaultProps = {
  staticContext: {},
};

ErrorPage.propTypes = {
  staticContext: PropTypes.shape({}),
};

export default ErrorPage;
