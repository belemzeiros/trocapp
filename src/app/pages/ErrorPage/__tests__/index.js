import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import ErrorPage from '../';

describe('ErrorPage', () => {
  const shallow = createShallow();

  it('ErrorPage will mount correctly', () => {
    const wrapper = shallow(<ErrorPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
