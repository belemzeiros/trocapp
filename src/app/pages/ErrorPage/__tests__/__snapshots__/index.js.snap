// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage ErrorPage will mount correctly 1`] = `
<React.Fragment>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Erro página não encontrada
    </title>
  </HelmetWrapper>
  <div>
    Página não existe!
  </div>
</React.Fragment>
`;
